### ISSUE TITLE
Missing dataset version history

### ISSUE THREAD

**Issue by AntoineD** [OpenLineage/OpenLineage#3303]
Hello,

I am learning open lineage, while doing the [tutorial with the python client](https://openlineage.io/docs/client/python#getting-started), I noticed that the job has 5 items in its run history and the output dataset has  5 items in its version history, which seems to be the expected behavior.

But the input dataset has only 1 item in its version history!

Is is expected? (I did not find an explanation neither in the doc, nor on github)
If yes, why?

Cheers.

**Comment by mobuchowski** [comment]
Yes - nothing has updated the input dataset in the meantime, so the dataset version is the same. Output datasets have different versions, since they are updated by the running job.

By the way, this is Marquez assumption - there is nothing preventing other systems to work other way.
